lib-injection/dl_wheels.py

```diff
@@ -73,34 +74,47 @@
 dl_dir = args.output_dir
 print("saving wheels to %s" % dl_dir)
 
-for python_version, arch, platform in itertools.product(args.python_version, args.arch, args.platform):
-    print("Downloading %s %s %s wheel" % (python_version, arch, platform))
-    abi = "cp%s" % python_version.replace(".", "")
-    # Have to special-case these versions of Python for some reason.
-    if python_version in ["2.7", "3.5", "3.6", "3.7"]:
-        abi += "m"
-
-    # See the docs for an explanation of all the options used:
-    # https://pip.pypa.io/en/stable/cli/pip_download/
-    #   only-binary=:all: is specified to ensure we get all the dependencies of ddtrace as well.
-    cmd = [
-        sys.executable,
-        "-m",
-        "pip",
-        "download",
-        "ddtrace==%s" % args.ddtrace_version,
-        "--platform",
-        "%s_%s" % (platform, arch),
-        "--python-version",
-        python_version,
-        "--abi",
-        abi,
-        "--only-binary=:all:",
-        "--dest",
-        dl_dir,
-    ]
-    if args.verbose:
-        print(" ".join(cmd))
-
-    if not args.dry_run:
-        subprocess.run(cmd, capture_output=not args.verbose, check=True)
+
+for python_version in args.python_version:
+    for arch, platform in itertools.product(args.arch, args.platform):
+        print("Downloading %s %s %s wheel" % (python_version, arch, platform))
+        abi = "cp%s" % python_version.replace(".", "")
+        # Have to special-case these versions of Python for some reason.
+        if python_version in ["2.7", "3.5", "3.6", "3.7"]:
```

Comment:
We don't technically need to since for the injection we only support >=3.7. Not much harm in keeping it though since the script works for the older versions.